src/types/account/index.ts

```diff
@@ -96,6 +98,12 @@ export type AccountSettingsAttrs = {
 	[key: string]: string | number | Array<string | number> | undefined;
 };
 
+export type UpdateSettings = (settingsMods: Partial<Mods>) => void;
+export type UpdateAccount = (
+	accountMods: Partial<Mods>,
+	response: Pick<EditSettingsBatchResponse, 'CreateIdentityResponse'>
+) => void;
```

Comment:
I feel the two function should not use the Mods or the EditSettingsBatchResponse utils, I would prefer having them implementing their own types, so that we can 
- give them better namings, without having to change all the internal implementation
- avoid to expose the setting types, which were created in a sort of migration from js, and are all but clean

The types could also be place directly near the function definitions imo, since the types folder is full of duplicated types and is not quite organized, and we are trying to dismantle it

Reply:
Should be better now